docs/binders.md

```diff
@@ -66,4 +66,14 @@ def foo(x: bit<10>, y: bit<10>) {
 }
 ```
 
+External functions, such as functions provided by header files, can be
+declared using the `extern` keyword.
+
+```
+def extern print_vector(c: float[4]);
+```
+
+Backends can either provide their implementations for `externs` (for example
```

Comment:
This is ridiculously pedantic, but:

    `extern`s

seems more correct than:

    `externs`

Reply:
fixed.